src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -1825,18 +1825,18 @@ public int Count
     {
         get
         {
-            return ((IReadOnlyList<int>)field).Count;
+            return ((IReadOnlyCollection<int>)field).Count;
         }
     }
 
     public IEnumerator<int> GetEnumerator()
     {
-        return ((IReadOnlyList<int>)field).GetEnumerator();
+        return ((IEnumerable<int>)field).GetEnumerator();
     }
 
     IEnumerator IEnumerable.GetEnumerator()
     {
-        return ((IReadOnlyList<int>)field).GetEnumerator();
+        return field.GetEnumerator();
```

Comment:
this is better as well. tehre was no need to actually call through an interface here as the field directly exposes the impl okf that interface member.